### PR TITLE
7.0 bill of lading webkit

### PR DESCRIPTION
**Pull request by mcassuto**
A Bill of Lading is a document issued by a carrier to a shipper of goods.
It is a negotiable instrument, and it serves three purposes:
- it is a receipt for the goods shipped;
- it evidences the contract of carriage;
- it serves as a document of title (i.e. ownership).

### CODE REVIEW

**Comment by bwrsandman** [comment]
`./bill_of_lading_webkit/report/stock_report.py:25:1: F401 'pooler' imported but unused`

**Comment by mcassuto** [comment]
@bwrsandman : I removed the import of pooler + I fixed also all the PEP8 violation from the other module to get travis successfull

**Comment by pedrobaeza** [comment]
Hi, following module naming conventions, I would call this module as _stock_bill_of_lading_webkit.

Regards.

**Comment by bwrsandman** [comment]
@mcassuto @pedrobaeza I fixed .travis.yml and added it to this PR

**Comment by mcassuto** [comment]
Hi @pedrobaeza ,

where can I find this naming convention ?
I would like to improve my commits and not having you to remind me this ;-)

**Comment by bwrsandman** [comment]
@mcassuto only thing causing failiure is:
`./stock_picking_webkit/report/stock_report.py:116:9: E265 block comment should start with '# '`

**Comment by pedrobaeza** [comment]
@mcassuto, thanks for your interest. That's something not written yet, because OCA conventions needs a review, but you can find this convention across all the core modules. The convention is more or less:

```
area_subarea_function
```

where

```
__area__ is the main area affected (stock, sale, mrp, purchase, account...)
__subarea__ is an object or subdivision reference. For example, inventory (for stock), invoice (for account), etc.
__function__ This is a free denomination that must be short and descriptive for the modification that the module is doing.
```

In your case, I think you don't need subarea, because the function (bill of lading) is generic enough.

Regards.

**Comment by mcassuto** [comment]
@bwrsandman : I fixed the Pep8 error, but it is a strange situation of having to fix errors on already published modules to get mine merged....
@pedrobaeza : thanks for the explanation , I renamed my module

**Comment by bwrsandman** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Hi, Marc, one last thing: please rename module name on the translations.

Thanks.

**Comment by bwrsandman** [comment]
Adding to that, the .pot file needs to be regenerated once it's renamed:
- [x] stock_bill_of_lading_webkit/i18n/bill_of_lading_webkit.pot -> stock_bill_of_lading_webkit/i18n/stock_bill_of_lading_webkit.pot
- [x] regenerate .pot file
- [x] Update fr.po file

**Comment by mcassuto** [comment]
hi @pedrobaeza ,

finally do I have your approval on this one ?

Regards,
Marc

**Comment by pedrobaeza** [comment]
Hi, Marc, please fix translations issue, and I'll get you my approval. Put a comment here to get notified when you do the change.

Regards.

**Comment by mcassuto** [comment]
@pedrobaeza : my very bad, I understood  Sandy did it.
Fixed !

**Comment by pedrobaeza** [comment]
Where is the .pot file?

**Comment by mcassuto** [comment]
@pedrobaeza here ;-)

**Comment by pedrobaeza** [comment]
Thanks.

:+1:

**Comment by bwrsandman** [comment]
Needs to update fr.po file

**Comment by pedrobaeza** [comment]
Are you sure? I see it correct.

File: stock_bill_of_lading_webkit/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* bill_of_lading_webkit
```
Comment (bwrsandman) [inline stock_bill_of_lading_webkit/i18n/fr.po]: not correct

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**